src/web/config_load.cc

```diff
@@ -296,10 +296,10 @@ void web::configLoad::process()
         pr++;
     }
 
-    std::vector<config_option_t> autoscanList = {
+    constexpr config_option_t autoscanList[] = {
```

Comment:
Could we use std::array here too?

Reply:
Part of the problem is that some of the members are ifdef'd out. std::array requires a size.

Reply:
Converted to std::array